src/pals/kinds/Quadrupole.py

```diff
@@ -11,5 +13,14 @@ class Quadrupole(ThickElement):
     kind: Literal["Quadrupole"] = "Quadrupole"
 
     # Quadrupole-specific parameters
+    MagneticMultipoleP: Optional[MagneticMultipoleParameters] = None
```

Comment:
I think for `Quadrupole` I made the parameters intentionally *not* optional, because it's a quad after all.
The original line 15 was thus better.

Reply:
Saw the [rest of the discussion](https://github.com/pals-project/pals-python/pull/49#issuecomment-3514136446) now. Ok with me to make both optional then.

Reply:
Electrostatic quadrupoles do exist so the standard should not preclude this.